src/image-copy/azext_imagecopy/custom.py

```diff
@@ -94,13 +94,15 @@ def imagecopy(cmd, source_resource_group_name, source_object_name, target_locati
                                        '--location', snapshot_location,
                                        '--resource-group', source_resource_group_name,
                                        '--source', source_os_disk_id,
-                                       '--source-storage-account-id', source_storage_account_id])
+                                       '--source-storage-account-id', source_storage_account_id,
+                                       '--hyper-v-generation', json_cmd_output['hyperVGeneration']])
```

Comment:
Can we add hyper_v_generation = json_cmd_output['hyperVGeneration'] after line 87
```suggestion
                                       '--hyper-v-generation', hyper_v_generation])
```

Reply:
OK, updated